docker-jans-shibboleth/scripts/bootstrap.py

```
@@ -0,0 +1,191 @@
+                f.write(encryption_cert)
+        
+        if encryption_key:
+            with open(f"{IDP_HOME}/credentials/idp-encryption.key", "w") as f:
+                f.write(encryption_key)
+    
+    def setup_webapp(self):
+        logger.info("Setting up webapp")
+        
+        war_src = os.environ.get(
+            "JANS_SHIBBOLETH_WAR",
+            f"/tmp/shibboleth-idp-src/war/idp.war"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)